dev/README.md

```diff
@@ -192,3 +192,29 @@ The script runs forever, on linux machines press `CTRL+C` to stop it (or just se
 With the parameters `--manager`, `--keycloak`, `--username` and `--password` you can configure the script so to target any environment (for example the demo environment).
 
 The grafana dashboards in the `grafana` folder are just for development purposes, but it's good to keep them in sync with the dashboards deployed in the demo environment (those dashboards are located at `kustomize/overlays/prod/observability/grafana`).
+
+## Knative
+
+The [knative-installer.sh](bin/knative-installer.sh) script applies Knative Eventing and Knative Eventing Kafka components on the Kubernetes cluster.
+
+Just run it without arguments:
+
+```bash
+./dev/bin/knative-installer.sh
+```
+
+## Knative Kafka Broker
+
+The [kafka-broker-setup.sh](bin/kafka-broker-setup.sh) script leverages the `MANAGED_KAFKA_INSTANCE_NAME` environment variable and creates a few Kubernetes resources:
+
+| Kubernetes Resource | Description                                              |
+|:--------------------|----------------------------------------------------------|
+| `Secret`            | Contains the service account credentials                 |
+| `ConfigMap`         | Reference to the `Secret` and link to the bootstrap host |
+| `Broker`            | Kafka broker, based on `ConfigMap` configuration         |
+
+Just run it without arguments:
+
+```bash
+./dev/bin/kafka-broker-setup.sh
+```
```

Comment:
Since this is not part of the setup but it's just to demo what the operator should do we might not include it in the README. 
But since we are targeting the `knative` branch we can have a final review when we move to `main`